src/index.js

```diff
@@ -1,40 +1,84 @@
 import './style.css';
 
```

Comment:
Please make sure you created a new file for the new tasks as mentioned on the project requirements.
<img width="775" alt="Screenshot 2023-01-29 at 1 19 44 AM" src="https://user-images.githubusercontent.com/25887253/215293658-2ce3402a-0d58-488d-92d2-f417575e8bf9.png">